src/main/java/com/solutions/done/NetPayableSalary.java

```diff
@@ -0,0 +1,42 @@
+package com.solutions.done;
+
+import java.util.Scanner;
+
+public class NetPayableSalary {
+
+    private static double basicSalary;
+    float $hra, $specialAllowance, $pf, $totalNetPayableSalary;
```

Comment:
Making this of type double is better so as to  to remove the unnecessary type casting